engine-rest/engine-rest-openapi/src/main/templates/paths/history/variable-instance/{id}/data/get.ftl

```diff
@@ -0,0 +1,81 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/variable-instance/get-variable-instance-binary/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricVariableInstanceBinary"
+      tag = "Historic Variable Instance"
+      summary = "Get Variable Instance (Binary)"
+      desc = "Retrieves the content of a historic variable by id. Applicable for variables that
+              are serialized as binary data."
+  />
+
+  "parameters" : [
+
+      <@lib.parameter
+          name = "id"
+          location = "path"
+          type = "string"
+          required = true
+          desc = "The id of the variable instance."
+          last = true
+      />
+
+  ],
+
+  "responses": {
+
+    "200": {
+      "description": "Request successful.",
+      "content": {
+        "application/octet-stream": {
+          "schema": {
+            "type": "string",
+            "format": "binary",
+            "description": "For binary variables or files without any MIME type information, a byte stream is returned."
+          },
+          "examples": {
+            "example-1": {
+              "summary": "GET `/history/variable-instance/someId/data`",
+              "value": "binary variable: Status 200. Content-Type: application/octet-stream"
+            }
+          }
+        },
+        "*/*": {
+          "schema": {
+            "type": "string",
+            "format": "binary",
+            "description": "File variables with MIME type information are returned as the saved type. Additionally,
+                            for file variables the Content-Disposition header will be set."
+          },
+          "examples": {
+            "example-1": {
+              "summary": "GET `/history/variable-instance/someId/data`",
+              "value": "file variable: Status 200. Content-Type: text/plain; charset=UTF-8.
+                        Content-Disposition: attachment; filename='someFile.txt'"
+            }
+          }
+        }
+      }
+    },
```

Comment:
We do have `multiTypeResponse` macro for responses with multiple media types on the same code. You can have a look at [DRD Diagram Request](https://github.com/camunda/camunda-bpm-platform/blob/459ddf901e8a305d2460645344d556266b430aa5/engine-rest/engine-rest-openapi/src/main/templates/paths/decision-requirements-definition/%7Bid%7D/diagram/get.ftl#L26) to see an example. This also shows how to use the `binary` attribute that takes care of creating the right `schema` just like you did manually here. 🙂